src/default_parameters.jl

```diff
@@ -7,15 +7,15 @@
     Tini=Tprog                # number format to reduce precision for initial conditions
 
     # DOMAIN RESOLUTION AND RATIO
-    nx::Int=100                         # number of grid cells in x-direction
-    Lx::Real=4000e3                     # length of the domain in x-direction [m]
-    L_ratio::Real=2                     # Domain aspect ratio of Lx/Ly
+    nx::Int=128                         # number of grid cells in x-direction
+    Lx::Int=3840e3                      # length of the domain in x-direction [m]
+    L_ratio::Int=1                      # Domain aspect ratio of Lx/Ly
```

Comment:
Why does this have to be an Int?

Reply:
I think we just didn't want a `Real` because of type-instabilities. Is there a different option to `Int` here?

Reply:
`Float64`?

Reply:
Also note that because you change the defaults this is a breaking change, because you said "works as before" at some point 😉